proposals/0107-unsaferawpointer.md

```diff
@@ -1477,34 +1477,36 @@ struct UnsafeMutableRawPointer : Strideable, Hashable, _Pointer {
   func bindMemory<T>(to: T.Type, capacity: Int) -> UnsafeMutablePointer<T>
   func assumingMemoryBound<T>(to: T.Type) -> UnsafeMutablePointer<T>
 
-  func initialize<T>(_: T.Type, with: T, count: Int = 1)
+  func initialize<T>(_: T.Type, to: T, count: Int = 1)
     -> UnsafeMutablePointer<T>
-  func initialize<T>(toContiguous: T.Type, atIndex: Int, with: T)
+  func initialize<T>(contiguous: T.Type, at: Int, to: T)
```

Comment:
Both of the above don’t look like they read well:

```swift
address.initialize(String.self, to: “hello")
address.initialize(contiguous: String.self, at: position, to: “hello”)
```
In the first one, it makes `String.self` read as the direct object, but we are not initializing the type; we’re initializing memory.

`contiguous` is very confusing in the latter call because it implies multiple elements, yet this initializes only a single element.  What’s it supposed to mean?